docker-angor-api/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+volumes:
+  blockchain:
+    name: btc-data
+
+services:
+  node:
+    container_name: btc-node
+    image: eugenenostrdev/bitcoin_signet:amd64
+    mem_limit: 2048m
+    cpus: 0.8
+    healthcheck:
+      test: ["CMD", "bitcoin-cli", "ping"]
+      interval: 10s
+      retries: 5
+      start_period: 5s
+      timeout: 10s
+    restart: unless-stopped
+    stop_grace_period: 15m
+    volumes:
+      - blockchain:/root/.bitcoin
+    environment:
+      UACOMMENT: $UACOMMENT
+      BLOCKPRODUCTIONDELAY: ${BLOCKPRODUCTIONDELAY:-30}
+      MINERENABLED: ${MINERENABLED:-0} # change this to 1 if this node needs to mine (however for simplicity there should only be one mining node per challenge)
+      NBITS: $NBITS
+      MINETO: ${MINETO:-tb1qk4pq0rh75qtph47wlufhyss43flhvhvwe4zt8a}
+      PRIVKEY: ${PRIVKEY:-cRz3Ci2aUNmdP4pViSM8LafwKHZmvn4X6gjeCXzVkBYBLhzA3uFC} # private key path is m/84'/1'/0'/0/0 of test wallet mnemonic "margin radio diamond leg loud street announce guitar video shiver speed eyebrow"
```

Comment:
this node is not gonna mine so we could remove this part but leave it there for now